yarn.lock

```diff
@@ -3284,6 +3294,24 @@ rdf-normalize@^1.0.0:
   resolved "https://registry.yarnpkg.com/rdf-normalize/-/rdf-normalize-1.0.0.tgz#53496baf362cce9d9fca1f2216c6c30007f99cca"
   integrity sha1-U0lrrzYszp2fyh8iFsbDAAf5nMo=
 
+rdf-parser-n3@^1.1.1:
+  version "1.1.1"
+  resolved "https://registry.yarnpkg.com/rdf-parser-n3/-/rdf-parser-n3-1.1.1.tgz#af748c6031676e245f21d84ddf0d21623cee7650"
+  integrity sha512-0fmITANt/DcOB9in1ZuVaLBDwmsaKhhe4MQMVovWE4VBA1Jw/6UHXrmJsB+DS5Gqa7KtQdIIFTNCbArJKujQcw==
+  dependencies:
+    n3 "^0.11.2"
+    rdf-data-model "^1.0.0"
+    rdf-sink "^1.0.0"
+    readable-stream "^2.2.9"
+
+rdf-sink@^1.0.0:
+  version "1.0.1"
+  resolved "https://registry.yarnpkg.com/rdf-sink/-/rdf-sink-1.0.1.tgz#e82f760233165f1cc7c58e55a1c6345644b31d78"
+  integrity sha512-AytWds7C3HdYOr0nLpLcqvaPnbNFMlZ+u3gccYPzISxBvByMIfMzKbMbCzjQTZVmnVMEfCzACm2+MXeASAACAg==
+  dependencies:
+    rdf-ext "^1.1.0"
+    readable-stream "^2.3.5"
```

Comment:
Those are some very old libraries. You should use `@rdfjs/parser-n3` and `@rdfjs/sink` instead

Reply:
Those were not necessary, I removed them, not sure why it was added, thanks !